src/dashboard/cd-dashboard-news.vue

```diff
@@ -47,6 +63,16 @@
         const res = await NewsForumsService.loadForums();
         this.forums = res.body.topics;
       },
+      sortPostsByDate(posts) {
+        const sortedPosts = posts.sort((a, b) =>
+         b.date - a.date);
+        return (sortedPosts.map(post => (Object.assign({
```

Comment:
Two things here:
1. We could avoid mapping over the posts again by adding a `formattedDate` property to `formattedNews` and `formattedForums`. Really, given the amount of data we have, there's very little performance impact of mapping again, but still...
2. As it stands, there's no need for `Object.assign` here. However, it would be useful in a different way
```javascript
return (sortedPosts.map(post => Object.assign({
  date: post.date.utc().format('DD/MM/YYYY'),
}, post);
```
This will create a new object with the formatted date, and then assign all properties from post to it that don't conflict (so date, for example, will not be copied over).

Reply:
Gotcha. Was looking into that but I got confused about not overwriting, makes so much sense. thanks